### PR TITLE
Torinnayak   task1

### PR DESCRIPTION
**Pull request by torinnayak1**
[//]: # "Feel free to customize this template to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Recreated the post, comments, and header from the figma

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: image

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
Please review the styles.module.css and page.tsx files 

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

flexbox froggy lmao

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @insert pl github username here

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,48 +1,205 @@
 import { FiHeart as HeartIcon } from 'react-icons/fi';
 import { GoPaperAirplane as ShareIcon } from 'react-icons/go';
-import { LuUserCircle as ProfileIcon } from 'react-icons/lu';
+//import { LuCircle as ProfileIcon } from 'react-icons/lu';
```
Comment (adityapawar1) [inline app/page.tsx:3]: You should be using
`import { LuUserCircle as ProfileIcon } from 'react-icons/lu';`
instead of pasting the svg info into the file.

File: app/page.tsx [hunk 2]
```diff
@@ -1,48 +1,205 @@
...
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="17"
+                  height="11"
+                  viewBox="0 0 17 11"
+                  fill="none"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M1 7.00263H2C2.55228 7.00263 3 7.45035 3 8.00263V10.0026C3 10.5549 2.55228 11.0026 2 11.0026H1C0.447715 11.0026 0 10.5549 0 10.0026V8.00263C0 7.45035 0.447715 7.00263 1 7.00263ZM5.66667 5.00263H6.66667C7.21895 5.00263 7.66667 5.45035 7.66667 6.00263V10.0026C7.66667 10.5549 7.21895 11.0026 6.66667 11.0026H5.66667C5.11438 11.0026 4.66667 10.5549 4.66667 10.0026V6.00263C4.66667 5.45035 5.11438 5.00263 5.66667 5.00263ZM10.3333 2.6693H11.3333C11.8856 2.6693 12.3333 3.11702 12.3333 3.6693V10.0026C12.3333 10.5549 11.8856 11.0026 11.3333 11.0026H10.3333C9.78105 11.0026 9.33333 10.5549 9.33333 10.0026V3.6693C9.33333 3.11702 9.78105 2.6693 10.3333 2.6693ZM15 0.335968H16C16.5523 0.335968 17 0.783683 17 1.33597V10.0026C17 10.5549 16.5523 11.0026 16 11.0026H15C14.4477 11.0026 14 10.5549 14 10.0026V1.33597C14 0.783683 14.4477 0.335968 15 0.335968Z"
```
Comment (adityapawar1) [inline app/page.tsx:94]: Instead of the svg source here, you should use `<ProfileIcon size={24} />`

File: app/page.tsx [hunk 3]
```diff
@@ -1,48 +1,205 @@
...
...99 3.34269 15.2994 3.25711C15.2989 3.17153 15.2644 3.08967 15.2033 3.02966C10.9882 -1.00989 4.33844 -1.00989 0.123342 3.02966C0.062283 3.08963 0.0276391 3.17146 0.0270761 3.25704C0.026513 3.34262 0.0600773 3.42489 0.120342 3.48566L1.27667 4.65233C1.3957 4.77223 1.58915 4.77372 1.71001 4.65566C3.31616 3.12876 5.44756 2.27732 7.66367 2.27733ZM7.66367 6.07299C8.88116 6.07292 10.0552 6.52545 10.9577 7.34266C11.0797 7.45864 11.272 7.45613 11.391 7.33699L12.5457 6.17033C12.6065 6.10913 12.6402 6.02612 12.6393 5.93985C12.6385 5.85359 12.603 5.77127 12.541 5.71133C9.79281 3.15494 5.53687 3.15494 2.78867 5.71133C2.7266 5.77127 2.69118 5.85363 2.69036 5.93992C2.68955 6.02621 2.72341 6.10922 2.78434 6.17033L3.93867 7.33699C4.05766 7.45613 4.24994 7.45864 4.37201 7.34266C5.27388 6.52599 6.44699 6.0735 7.66367 6.07299ZM9.88267 8.85733C9.94443 8.79672 9.97844 8.71332 9.97667 8.62681C9.9749 8.54031 9.93752 8.45836 9.87334 8.40033C8.59777 7.32144 6.72958 7.32144 5.45401 8.40033C5.38979 8.45831 5.35234 8.54023 5.35051 8.62674C5.34868 8.71325 5.38263 8.79668 5.44434 8.85733L7.44201 10.873C7.50056 10.9322 7.58038 10.9656 7.66367 10.9656C7.74697 10.9656 7.82679 10.9322 7.88534 10.873L9.88267 8.85733Z"
```
Comment (adityapawar1) [inline app/page.tsx:110]: Same here with the svg

File: app/page.tsx [hunk 4]
```diff
@@ -1,48 +1,205 @@
...
+                      d="M8.93436 11H22.8267V13H8.93436L15.0565 18.364L13.4426 19.778L4.56534 12L13.4426 4.22198L15.0565 5.63598L8.93436 11Z"
+                      fill="#4E4B66"
+                    />
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_3_2152">
+                      <rect width="27.392" height="24" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg>
```
Comment (adityapawar1) [inline app/page.tsx:168]: Replace svg with corresponding react-icons image from image from https://react-icons.github.io/react-icons/icons/lu/

File: app/styles.module.css [hunk 5]
```diff
@@ -5,12 +5,181 @@
...
+}
+
+.name {
+  /* display:flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start; */
+  font-weight: 500;
+  margin-left: 5px;
+  font-size: 85%;
```
Comment (adityapawar1) [inline app/styles.module.css:51]: Use exact pixel font size from figma (i believe this one is 13px)

File: app/page.tsx [hunk 6]
```diff
@@ -1,48 +1,205 @@
...
+          <ShareIcon size={24} />
+        </span>
 
-        <ProfileIcon size={24} />
-        <p>daviddd</p>
-        <p>September 20</p>
-        <p>
+        <div className={styles.profile}>
+          <div className={styles.picName}>
+            <div className={styles.avatar} />
```
Comment (adityapawar1) [inline app/page.tsx]: Avoid using empty divs for shapes. You should be using the `<ProfileIcon size={24} />`, imported from react-icons.

File: app/page.tsx [hunk 7]
```diff
@@ -1,48 +1,205 @@
...
-        </p>
+        </div>
 
-        <ProfileIcon size={24} />
-        <p>vppraggie</p>
-        <p>September 21</p>
-        <p>Thanks for sharing!</p>
+        <div className={styles.profile}>
+          <div className={styles.picName}>
+            <div className={styles.avatar} />
```
Comment (adityapawar1) [inline app/page.tsx]: Same here, use a the `ProfileIcon` instead of a div.

File: app/styles.module.css [hunk 8]
```diff
@@ -5,12 +5,181 @@
...
+  font-size: 11px;
+  align-self: last baseline;
+}
+
+.words {
+  margin: 15px;
+  font-size: 13px;
+}
+
+.pic {
```
Comment (adityapawar1) [inline app/styles.module.css:74]: Use a  (or <Image> from 'next/image') to render images, instead of using CSS.

File: app/page.tsx [hunk 9]
```diff
@@ -1,48 +1,205 @@
...
-          https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg
-        </p>
-
-        <HeartIcon size={24} />
-        <p>256 Likes</p>
-        <ShareIcon size={24} />
+          https:// www.codetenderloin.org/
+        </div>
+        <div className={styles.pic}></div>
+        <span className={styles.withLine}>
```
Comment (adityapawar1) [inline app/page.tsx]: I would make this line just an empty div, and separate the flex below since they aren't really related.

File: app/styles.module.css [hunk 10]
```diff
@@ -5,12 +5,181 @@
...
+  margin-left: 5px;
+  font-size: 85%;
+  align-self: center;
+}
+
+.avatar {
+  width: 22px;
+  height: 22px;
+  background-color: #ccc;
+  border-radius: 50%;
```
Comment (adityapawar1) [inline app/styles.module.css:59]: Avoid empty divs with just CSS styling

**Review by adityapawar1 — CHANGES_REQUESTED** [review]
Looks great! Couple of small changes!!
